### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by bogem**

### CODE REVIEW

File: README-ruRU.md [hunk 1]
```diff
@@ -2102,7 +2102,7 @@
     include AnotherModule
 
     # вложенные классы
-    CustomErrorKlass = Class.new(StandardError)
+    CustomErrorClass = Class.new(StandardError)
```
Comment (arbox) [inline README-ruRU.md]: It is not an error, it is a good practice to avoid language keywords in identifiers, particularly to write Klass instead of Class.

Comment (bogem) [inline README-ruRU.md]: Okay. I didn't know about it

**Comment by arbox** [comment]
Ok, @bogem , please change this PR, I'll merge it then.

**Comment by bogem** [comment]
@arbox PR? What it is?

**Comment by arbox** [comment]
PR = pull request

**Comment by bogem** [comment]
Oh :) Sorry, I'm newbie in GitHub
Ok. I will do it

**Comment by bogem** [comment]
@arbox Done

**Comment by arbox** [comment]
Thank you, @bogem , merged! Good work!

**Comment by bogem** [comment]
@arbox Thank you :)

**Pull request merged**